sipi/config/sipi.docker-config.lua

```diff
@@ -195,6 +195,11 @@ routes = {
         method = 'DELETE',
         route = '/delete_temp_file',
         script = 'delete_temp_file.lua'
+    },
+    {
+        method = 'POST',
+        route = '/upload_without_transcoding',
```

Comment:
In DSP-TOOLS, I always refer to this as "preprocessing", rather than "transcoding". For me, "preprocessing" is more telling than "transcoding", but perhaps this is just me.

Reply:
To me, transcoding doesn't sound natural, either. But it comes from Ivan. @subotic , do you have a specific reason or strong opinion about this? I wouldn't call it "preprocessing", because it is not prior to something. But just "processing".

Reply:
Nope, you can call it however you like. Preprocessing / processing both sounds fine to me. I would lean towards `processing` just because it is shorter.